ui/src/Pages/LoginPage/LoginPage.tsx

```diff
@@ -96,12 +96,7 @@ const LoginForm = () => {
         className="w-full"
         type="password"
         label={t('login_page.field.password', 'Password')}
-        {...register('password', { required: true })}
-        error={
-          formErrors.password?.type == 'required'
-            ? 'Please enter a password'
-            : undefined
-        }
+        {...register('password', { required: false })}
```

Comment:
I don't think this register is required now that it's false... I'd imagine this would be the default right?